client/components/site-header.tsx

```diff
@@ -2,52 +2,82 @@
 
 import Link from "next/link"
 import { useRouter } from "next/navigation"
+import { useI18n } from "@/utils/services/internationalization/client"
 
 import { siteConfig } from "@/config/site"
 import useUserStore from "@/hooks/state/user/store"
 import { Button } from "@/components/ui/button"
-import UserDropDownMenu from "@/components/customized-ui/drop-down-menu/user"
+import BaseAvatar from "@/components/customized-ui/avatars/base"
+import LocaleDropDownMenu from "@/components/customized-ui/drop-down-menu/locale"
+import MessageIcon from "@/components/customized-ui/icons/message"
 import { BaseNavigationMenu } from "@/components/customized-ui/navigation-menu/base"
+import { MobileNavigationMenu } from "@/components/customized-ui/navigation-menu/mobile"
 import { Icons } from "@/components/icons"
 import { MainNav } from "@/components/main-nav"
 import { ThemeToggle } from "@/components/theme-toggle"
 
 export function SiteHeader() {
-  const isUserSignIn = useUserStore((state) => state.isSignIn)
+  const t = useI18n()
+  const user = useUserStore((state) => state)
   const router = useRouter()
 
   return (
     <header className="sticky top-0 z-40 w-full border-b bg-background">
-      <div className="container flex h-16 items-center justify-center space-x-4 sm:space-x-0">
-        <div className="flex-1">
-          <MainNav />
-        </div>
+      <div className="flex h-16 items-center justify-between space-x-4 p-2 sm:space-x-0  md:p-4">
+        <div className="flex items-center">
+          <MobileNavigationMenu className="mr-1 md:hidden" />
 
-        <div className="hidden flex-1 justify-center md:flex">
+          <div className="flex-1">
+            <MainNav />
+          </div>
+        </div>
+        <div className="hidden w-full flex-1 shrink-0 justify-center md:flex">
           <BaseNavigationMenu />
         </div>
 
-        <div className="flex flex-1 items-center justify-end space-x-4">
+        <div className="flex items-center justify-end space-x-4">
           <nav className="flex items-center gap-2">
-            <ThemeToggle />
-            <Link
-              aria-label="referalah-project-github"
-              href={siteConfig.links.github}
-              target="_blank"
-              rel="noreferrer"
-            >
-              <Icons.github />
-            </Link>
+            <LocaleDropDownMenu />
+            {
+              <div className="hidden md:block">
+                <ThemeToggle />
+              </div>
+            }
+            {
+              <Link
+                aria-label="referalah-project-github"
+                href={siteConfig.links.github}
+                target="_blank"
+                rel="noreferrer"
+                className="hidden md:block"
+              >
+                <Icons.github />
+              </Link>
+            }
```

Comment:
```suggestion
            <Link
              aria-label="referalah-project-github"
              href={siteConfig.links.github}
              target="_blank"
              rel="noreferrer"
              className="hidden md:block"
            >
              <Icons.github />
            </Link>
```

Same here, this curly bracket can be removed too!

Reply:
Updated please check